webapp/sass/layout/_sidebar-left.scss

```diff
@@ -206,6 +206,9 @@
     }
 
     @import "../components/status-icon";
+    .status-wrapper {
+        z-index: 999999;
```

Comment:
Is this necessary? I tend to worry when I see arbitrarily large z indexes since it's probably going to overlap something unintended like a modal dialog